### DISCUSSION TITLE
Creation d'une sous-boutique SANDBOX

### DISCUSSION THREAD

**Discussion by thefox82** [PayGreen/documentation#3] [Q&A] [answered]
Bonjour,
Je voulais savoir comment activer le mode marketplace dans la SANDBOX et comment créer des sous-boutiques?
Merci

**Comment by IMP3ter** [comment, marked as answer]
Bonjour,

Si vous avez créé un compte PayGreen en mode MarketPlace, votre sandbox sera en mode MarketPlace également.
Pour créé des sous-boutiques vous pouvez utiliser votre back-office ou notre API https://developers.paygreen.fr/docs/shop-create

Si vous rencontrez un problème avec votre compte, vous pouvez ouvrir un ticket sur notre support client.

Bonne journée,

**Reply by Githamza** [reply]
Bonjour , je ne me rappelle plus si j'ai crée la boutique en mode marketplace ou non ? est ce qu'il y a un moyen de le savoir ? sinon comment on peut switcher ?
mail ( hhamza@hmz-digital.fr)

Merci
